### PR TITLE
Added meta tag for homepage and appled h1 and h2 changes in homepage.

### LINKED ISSUE #4: Fix meta tags
There are currently several meta tags outdated from the original blockscout repo.

Such as:

```<meta name="description" content="BlockScout provides analytics data, API, and Smart Contract tools for the Sokol">

<meta name="keywords" content="BlockScout, blockchain, api, analytics, address, smart contract">```

### PR DESCRIPTION
**Pull request by ysong42**
Screenshot 2021-07-30 at 12 33 52 PM

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex [hunk 1]
```diff
@@ -175,7 +175,7 @@
   <div class="card card-chain-blocks">
     <div class="card-body">
       <%= link(gettext("View All Blocks"), to: block_path(BlockScoutWeb.Endpoint, :index), class: "btn-line float-right") %>
-      <h2 class="card-title"><%= gettext "Blocks" %></h2>
+      <h1 class="card-title"><%= gettext "Blocks" %></h2>
```
Comment (cdc-william) [inline apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex]: This block should end with `h1`

```suggestion
      <h1 class="card-title"><%= gettext "Blocks" %></h1>
```

File: apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex [hunk 2]
```diff
@@ -233,7 +233,7 @@
   <div class="card card-chain-transactions">
     <div class="card-body">
       <%= link(gettext("View All Transactions"), to: transaction_path(BlockScoutWeb.Endpoint, :index), class: "btn-line float-right") %>
-      <h2 class="card-title lg-card-title"><%= gettext "Transactions" %></h2>
+      <h1 class="card-title lg-card-title"><%= gettext "Transactions" %></h2>
```
Comment (cdc-william) [inline apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex]: Same here :)

```suggestion
      <h1 class="card-title lg-card-title"><%= gettext "Transactions" %></h1>
```

**Comment by cdc-william** [comment]
Apart from the 2 Minor edits, other are looking good to me, thanks!

**Comment by ysong42** [comment]
This PR fixed this issue: https://github.com/crypto-org-chain/cronos-blockscout/issues/4

**Comment by crypto-matto** [comment]
I believe you may try to remove `--warnings-as-errors` in `cronos-blockscout/apps/explorer/mix.exs` => `defp env_aliases(_env)`

```
defp env_aliases(_env) do
    [compile: "compile"]
end
```

**Pull request merged**